client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/style.scss

```diff
@@ -272,17 +275,11 @@ $font-family: "SF Pro Text", $sans;
 		scrollbar-gutter: stable;
 	}
 
-	.screen-container__body--no-padding {
-		padding: 0;
-	}
-
-	.screen-container__body--align-sides {
-		// Reduce the padding bottom of .navigator-header__description
-		margin-top: -8px;
+	.screen-container__body--extra-padding {
```

Comment:
I think, instead of adding this class to multiple screens, we can do the following:

- Explicitly add `margin-bottom: 32px` to `.navigator-header`
- Removed `padding-bottom` from `navigator-header__description`

This way we can see explicitly that `<NavigatorHeader>` has margin bottom of 32 px, not 14px + 18px.

What do you think?

Reply:
Looks like the space between the header and content is at least 32px for all screens. I agreed that we can simply add `margin-bottom` to the header 🙂

Reply:
Good idea!

- it's 30px not 32px because we have 2px already for the border on focus.
- we need a negative margin for the categories screen because it's reduced

Reply:
I just pushed the [changes](https://github.com/Automattic/wp-calypso/pull/78587/commits/d8975db97cb21b9e588fddf20b0f74b132367b53). I'll merge after the tests pass.

Reply:
> we need a negative margin for the categories screen because it's reduced

Oh, that's unfortunate, but anyway it's still better than nothing 😄 🚢 🚀